js/samples/04.ai.d.chainedActions.listBot/src/prompts/summarize/skprompt.txt

```diff
@@ -53,7 +56,12 @@ Here's the users query:
 ```
 
 Use the template to create an adaptive card.
+Create a Container for each list.
+Replace the first TextBlock text in the Container with the List name.
+Replace the second TextBlock text in the Container with the list items.
 Take into account any changes asked for by the user.
-Always replace the placeholder text, List n and Item n, with real data.
+-----------------------------------------------------------------------
```

Comment:
I think you should be able to get the same behavior without the "-" delimiter. These are extra unnecessary tokens.

Reply:
Updating the file with a single dash. Thanks for the suggestion :)